client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementSubmitter.vue

```diff
@@ -0,0 +1,284 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <fieldset data-dp-validate="statementSubmitterData">
+    <legend
+      id="submitter"
+      class="mb-3 color-text-muted font-normal">
+      {{ Translator.trans('submitted.author') }}
+    </legend>
+
+    <!--  TO DO: add if not participationGuestOnly  -->
+    <div
+      v-if="hasPermission('field_statement_meta_orga_name')"
+      aria-labelledby="submitter"
+      class="mb-2">
+      {{ submitterRole }}
+    </div>
+
+    <div class="grid grid-cols-1 gap-x-4 md:grid-cols-2">
+      <dp-input
+        v-if="hasPermission('field_statement_meta_orga_department_name') && !this.localStatement.attributes.isSubmittedByCitizen"
+        id="statementDepartmentName"
+        v-model="localStatement.attributes.initialOrganisationDepartmentName"
+        class="mb-2"
+        :disabled="!editable || !isStatementManual"
+        :label="{
+          text: Translator.trans('department')
+        }" />
+
+      <!--  TO DO: add if not participationGuestOnly -->
+      <dp-input
+        v-if="!this.localStatement.attributes.isSubmittedByCitizen"
+        id="statementOrgaName"
+        v-model="localStatement.attributes.initialOrganisationName"
+        class="mb-2"
+        :disabled="!editable || !isStatementManual"
+        :label="{
+          text: Translator.trans('organisation')
+        }" />
+
+      <dp-contextual-help
+        v-if="isSubmitterAnonymized()"
+        class="float-right mt-0.5"
+        :text="submitterHelpText" />
+      <dp-input
+        v-if="hasPermission('field_statement_meta_submit_name') && this.statementFormDefinitions.name.enabled"
+        id="statementSubmitterName"
+        v-model="statementSubmitterValue"
+        class="mb-2"
+        :disabled="!isStatementManual || !editable || isSubmitterAnonymized()"
+        :label="{
+          text: Translator.trans('name')
+        }" />
+
+      <dp-input
+        v-if="hasPermission('field_statement_submitter_email_address') || isStatementManual"
+        id="statementEmailAddress"
+        v-model="localStatement.attributes.submitterEmailAddress"
+        class="mb-2"
+        :disabled="!editable || !isStatementManual"
+        :label="{
+          text: Translator.trans('email')
+        }"
+        type="email" />
+
+      <dp-input
+        v-if="localStatement.attributes.represents"
+        id="statementRepresentation"
+        disabled
+        :label="{
+          text: Translator.trans('statement.representation.assessment')
+        }"
+        :value="localStatement.attributes.represents" />
+<!--      // TO DO: DpCheckbox?-->
```

Comment:
can this TODO be removed?

Reply:
Yes, after changing the input to checkbox. Thanks! [6424e2a](https://github.com/demos-europe/demosplan-core/pull/3896/commits/6424e2a6fd0e88b816aee0cbbac2f89062ea2162)

Reply:
Accidentally deleted the v-if condition and added here again [f449569](https://github.com/demos-europe/demosplan-core/pull/3896/commits/f44956984ab559bb82b7ed249a68838ca321ced3)